package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "foreman_leapp",
+  "version": "0.0.1",
+  "description": "Plugin for RHEL upgrades using Foreman.",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "node node_modules/.bin/jest webpack",
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/"
+  },
```

Comment:
can you add those that also comes with @theforeman packages:
"scripts": {
    "lint": "tfm-lint --plugin -d /webpack",
    "test": "tfm-test --plugin",
    "test:watch": "tfm-test --plugin --watchAll",
    "test:current": "tfm-test --plugin --watch",
    "publish-coverage": "tfm-publish-coverage",
    "stories": "tfm-stories --plugin",
    "stories:build": "tfm-build-stories --plugin",
    "stories:deploy": "surge --project .storybook-dist"
  },